options.html

```diff
@@ -136,6 +136,37 @@
 {
   box-shadow: 0px 0px 10px #e03030;
 }
+#userFilterLists {
+    margin-bottom: 8px;
+}
+
+#whitelistForm #newWhitelistDomain,
+#customFilterForm #newFilter,
+#subscriptionSelector,
+#customSubscriptionTitle,
+#customSubscriptionLocation
```

Comment:
There's a comma missing at the end of this selector.

Reply:
I agree.